src/test/java/codeu/model/store/persistence/PersistentDataStoreTest.java

```diff
@@ -146,4 +147,49 @@ public void testSaveAndLoadMessages() throws PersistentDataStoreException {
     Assert.assertEquals(contentTwo, resultMessageTwo.getContent());
     Assert.assertEquals(creationTwo, resultMessageTwo.getCreationTime());
   }
+
+    @Test
+    public void testSaveAndLoadActivities() throws PersistentDataStoreException {
+        UUID idOne = UUID.fromString("10000000-2222-3333-4444-555555555555");
+        String descriptionOne = "User1 has joined!";
+        Instant datetimeOne = Instant.ofEpochMilli(1);
```

Comment:
`ofEpochMilli` retrieves the EPOCH instant, and sets the number of milliseconds using the passed argument